### ISSUE TITLE
gen-api-gtkdoc.py autotools check broken?

### ISSUE THREAD

**Issue by eht16** [geany/geany#929]
On Windows, I just built Geany and it failed because I had python-lxml not installed in MSYS2. Though even the configure check failed with an ImportError, `make` still tried to use the Python script.

Log output: http://pastebin.geany.org/TWdfm/

I'm wondering about:
```
checking for python lxml package... Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named lxml
no
```
but then later on `configure` says: `Generate GtkDoc header           : yes`.

Installing python-lxml is no problem, will do this but still wondering about failing compilation.
Either it is something Windows specific or the configure check might be wrong.

@kugel-  ideas?

**Comment by codebrainz** [comment]
I thought https://github.com/codebrainz/geany/commit/02c39a3c1f5132cc39fba112e55df7ecf1a68bba would've done it. Is it even applied? I can't keep track of all the history re-writing, but I think if my [PR](https://github.com/kugel-/geany/pull/5) was merged, it would show a purple "merged" graphic.

@eht16 did you maybe forget to re-run `autogen.sh` to re-generate the `configure` script?

**Comment by kugel-** [comment]
@codebrainz there was no rewrite. I cherry picked your commits. It should be applied, although with a different hash

**Comment by codebrainz** [comment]
@kugel- ok, I guess I didn't look far enough back in the log. Out of curiosity, why cherry-pick instead of merging my PR branch into yours?

**Comment by b4n** [comment]
Probably my mistake somehow, I'll check that

**Comment by b4n** [comment]
OK, blunt mistake from my part in handling of =auto, fixed in 78c6a2ad6dfafb206024e828cb76ec3229768ab4.
Great you noticed, thanks!

**Comment by eht16** [comment]
Thanks for the fix, works like a charm now, with and without python-lxml.
